static_src/components/dropdown.jsx

```diff
@@ -0,0 +1,29 @@
+
+import React from 'react';
```

Comment:
I really like how you break out these components. I'm guessing it will be easier this way also once we move to server side rendering?